seismic/traveltime/requirements_conda_py37.txt

```diff
@@ -0,0 +1,28 @@
+#(hiperseispy37) @vdi-n23
+# conda install --file requirements_conda_py37.txt
+
+sphinx>=1.8
+numpy==1.16.4
```

Comment:
For hard pinning like this, it's a good idea to add a comment to explain why it is pinned. Otherwise `>=` is better.